tjpcov/covariance_cluster_mass.py

```diff
@@ -60,9 +60,9 @@ def load_from_cosmology(self, cosmo):
             cosmo (:obj:`pyccl.Cosmology`): Input cosmology
         """
         self.cosmo = cosmo
-        mass_def = ccl.halos.MassDef200m()
+        mass_def = ccl.halos.MassDef200m
         self.c = ccl.physical_constants.CLIGHT / 1000
-        self.mass_func = ccl.halos.MassFuncTinker08(cosmo, mass_def=mass_def)
+        self.mass_func = ccl.halos.MassFuncTinker08(mass_def=mass_def)
```

Comment:
In test_covariance_clusters.py [here](https://github.com/LSSTDESC/TJPCov/blob/346c189c11800cdc3c0d216154af0f4bb2f64734/tests/test_covariance_clusters.py#L125) it is used Tinker10 . @mattkwiecien please check if it is ok